crates/languages/src/json.rs

```diff
@@ -244,3 +251,123 @@ fn schema_file_match(path: &Path) -> &Path {
     path.strip_prefix(path.parent().unwrap().parent().unwrap())
         .unwrap()
 }
+
+pub(super) struct NodeVersionAdapter;
+
+#[async_trait(?Send)]
+impl LspAdapter for NodeVersionAdapter {
+    fn name(&self) -> LanguageServerName {
+        LanguageServerName("package-version-server".into())
+    }
+
+    async fn fetch_latest_server_version(
+        &self,
+        delegate: &dyn LspAdapterDelegate,
+    ) -> Result<Box<dyn 'static + Send + Any>> {
+        let release = latest_github_release(
+            "zed-industries/package-version-server",
+            true,
+            false,
+            delegate.http_client(),
+        )
+        .await?;
+        let os = match consts::OS {
+            "macos" => "apple-darwin",
+            "linux" => "unknown-linux-gnu",
+            "windows" => "pc-windows-msvc",
+            other => bail!("Running on unsupported os: {other}"),
+        };
+        let asset_name = format!("package-version-server-{}-{os}.tar.gz", consts::ARCH);
```

Comment:
On windows, I guess this is `.zip` file?
![Screenshot 2024-06-25 161749](https://github.com/zed-industries/zed/assets/14981363/5808ff1e-87cf-4120-9d28-1e8317d08f38)

Reply:
@JunkuiZhang Yup, I've fixed that up. I have no way of testing this though, as I don't have Zed set up on my Windows machine.